src/curated-corpus/pages/SchedulePage/SchedulePage.tsx

```diff
@@ -347,7 +348,7 @@ export const SchedulePage: React.FC = (): ReactElement => {
     const variables = {
       externalId: item.externalId,
       scheduledDate: item.scheduledDate,
-      source: ScheduledItemSource.Manual,
+      source: item.source,
```

Comment:
:+1: When an ML scheduled item is moved to the bottom, it should preserve the ML label.